### PR TITLE
TDP-3992 Add to/from numbers and updated conference name pattern

### PR DESCRIPTION
**Pull request by mrmeyers99**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3992.s3-website-us-east-1.amazonaws.com/

File: voice/bxml/callbacks/conferenceMemberExit.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ HTTP/1.1 204
 | conferenceId | The id for the conference that was joined                                                                       |
 | name         | The custom name used to reference this conference.                                                              |
 | callId       | The callId of the member that left the conference.                                                              |
+| from         | The from number from the original call                                                                          |
+| to           | The to number from the original call                                                                            |
```
Comment (muesono-daitan) [inline voice/bxml/callbacks/conferenceMemberExit.md:18]: Please also update the example below.

We could also use a more realistic conferenceId, like:
conf-59082d52-4a2ab5be-ce26-43ed-af94-431b8a19d4e3

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3992.s3-website-us-east-1.amazonaws.com/

**Review by muesono-daitan — APPROVED** [review]

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-TDP-3992.s3-website-us-east-1.amazonaws.com/

**Review by vsmith-bw — APPROVED** [review]

**Pull request merged**